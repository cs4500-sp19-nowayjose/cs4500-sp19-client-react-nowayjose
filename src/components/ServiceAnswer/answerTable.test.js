import React from 'react'
import AnswerTable from './answerTable'
import renderer from 'react-test-renderer'
import serviceAnswers from '../../data/serviceAnswers.mock.json'
import {BrowserRouter as Router} from 'react-router-dom'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
      <AnswerTable
        pageState={{ page: 0, resultsPerPage: serviceAnswers.length }}
        serviceAnswers={serviceAnswers}
        deleteAnswer={(id) => {}}
        />
      </Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
})
