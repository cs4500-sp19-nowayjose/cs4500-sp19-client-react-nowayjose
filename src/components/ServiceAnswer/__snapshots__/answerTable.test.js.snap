// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<table
  className="table"
>
  <thead>
    <tr>
      <th>
        User ID
      </th>
      <th>
        Service Question ID
      </th>
      <th>
        Choice Answer
      </th>
      <th>
        T/F
      </th>
      <th>
        Min Range
      </th>
      <th>
        Max Range
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td
        align="center"
      >
        1
      </td>
      <td
        align="center"
      >
        1
      </td>
      <a
        href="/admin/service-answers/1"
        onClick={[Function]}
      >
        <td
          align="center"
        />
        <td
          align="center"
        />
        <td
          align="center"
        >
          25000
        </td>
        <td
          align="center"
        >
          600
        </td>
      </a>
      <span
        onClick={[Function]}
      >
        X
      </span>
    </tr>
    <tr>
      <td
        align="center"
      >
        1
      </td>
      <td
        align="center"
      >
        4
      </td>
      <a
        href="/admin/service-answers/2"
        onClick={[Function]}
      >
        <td
          align="center"
        >
          0
        </td>
        <td
          align="center"
        />
        <td
          align="center"
        />
        <td
          align="center"
        />
      </a>
      <span
        onClick={[Function]}
      >
        X
      </span>
    </tr>
  </tbody>
</table>
`;
